deploy/olm-catalog/bundle/manifests/migration.openshift.io_migplans.yaml

```diff
@@ -94,8 +96,10 @@ spec:
                   the desired phase to run it in
                 properties:
                   executionNamespace:
+                    description: ExecutionNamespace holds the name of the namespace where hooks should be implemented.
```

Comment:
I think "where the hooks will be executed" would be more accurate here.

Reply:
Done